exotic/api/filters.py

```diff
@@ -43,6 +43,9 @@
         "PanSTARRS Y": {"name": "Y", "fwhm": ("946.4", "1054.4")},
         "PanSTARRS w": {"name": "N/A", "fwhm": ("404.2", "845.8")},
 
+        "Clear (unfiltered) reduced to V sequence": {"name": "CV", "fwhm": (None, None)},
+        "Clear with blue-blocking": {"name": "CBB", "fwhm": (None, None)},
+
```

Comment:
These would be the only values in the known list without definitive FWHM values. That sort of makes it hard to use as a definitive reference. 

If you would like to use the names, I think we should either enter the user-set description and keep their entered filter values. I think the matching can spiral as there all kinds of paths that follow from heuristics. Heuristics is really a road to perdition. 

Another concern I have about CBB is leaving the fwhm values open ended. I believe `blue-blocking` implies specific wavelengths, so if we leave that open ended, someone could enter red blocking values or anything to make the text meaningless. 

On the CITISENS side I am dealing with all kinds of inputs from various platforms, including AstroImageJ, which does not do this kind of validation AFAIK. But there could still be the same thing done manually in EXOTIC. ... I guess I need to chat to understand the impetus for what is being attempted.